bin/deploy_assets

```diff
@@ -5,4 +5,5 @@ set -e
 git checkout master
 git pull
 ./bin/version_check
+gulp assets-build
```

Comment:
There's still a task in there doing something weird:
https://github.com/everydayhero/hui/blob/dh-fix-deploy/assets.js

`assets-build` bundles that file. The demo page works because the task calls `index` which depends on `scripts` (same as dev).

We should perhaps bump the priority of working on how HUI is distributed? We'd have less trouble with testing setups if HUI didn't need to be compiled when it's included as a module.